testing/base/source_gen.h

```diff
@@ -163,8 +230,9 @@ class SourceGen {
   // `GetShuffledIdentifiers`.
   //
   // Usually, benchmarks should use the shuffled version. However, this is
-  // useful when there is already a post-processing step to shuffle things as it
-  // is *dramatically* more efficient, especially in debug builds.
+  // useful when the identifiers will be used in multiple different ways and
+  // only then shuffled, or if there is already a post-processing step to
+  // shuffle things which is very expensive in debug builds.
```

Comment:
```suggestion
  // useful when deterministic access to the identifiers is needed to avoid
  // introducing noise, or if there is already a post-processing step to shuffle
  // things, since shuffling is very expensive in debug builds.
```